Source/VCExpr/Boogie2VCExpr.cs

```diff
@@ -397,14 +397,15 @@ private class VariableMapping<VarKind> : ICloneable {
           BinaryOperator oper = (BinaryOperator)node.Fun;
           if (oper.Op == BinaryOperator.Opcode.Imp)
               flipContextForArg0 = true;
-          else if (oper.Op == BinaryOperator.Opcode.Iff) {
-              Expr one = new NAryExpr(node.tok, new BinaryOperator(node.tok, BinaryOperator.Opcode.Imp), new List<Expr> { node.Args[0], node.Args[1] });
-              Expr two = new NAryExpr(node.tok, new BinaryOperator(node.tok, BinaryOperator.Opcode.Imp), new List<Expr> { node.Args[1], node.Args[0] });
-              NAryExpr cmpd = new NAryExpr(node.tok, new BinaryOperator(node.tok, BinaryOperator.Opcode.And), new List<Expr> { one, two });
-              TypecheckingContext tc = new TypecheckingContext(null);
-              cmpd.Typecheck(tc);
-              return TranslateNAryExpr(cmpd);
-          }
+          // This code is no longer needed - left over from when SMT-LIB didn't support Boolean equality
```

Comment:
Just delete it.

Reply:
OK - for clarity I've put this in a separate PR.